.github/scripts/notifyWhenSpikeComplete.js

```diff
@@ -0,0 +1,70 @@
+// When the "spike complete" label is added to an issue:
+// 1. Modifies the labels,
+// 2. Updates the assignees and milestone, and
+// 3. Generates a notification to the Calcite project manager(s)
+//
+// The secret is formatted like so: person1, person2, person3
+//
+// Note the script automatically adds the "@" character in to notify the project manager(s)
+module.exports = async ({ github, context }) => {
+  const { managers } = process.env;
+  const { label } = context.payload;
+
+  if (label && label.name === "spike complete") {
+    // Add a "@" character to notify the user
+    const calcite_managers = managers.split(",").map((v) => " @" + v.trim());
+
+    const issueProps = {
+      issue_number: context.issue.number,
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+    };
+
+    /* Modify labels */
+
+    /* Tries to remove labels */
+    /* If the label is not associated with the issue,
+       the error is logged and the script will continue. */
+
+    // Try to remove "spike" label
+    try {
+      await github.rest.issues.removeLabel({
+        ...issueProps,
+        name: "spike",
+      });
+    } catch (err) {
+      console.log("The 'spike' label is not associated with the issue", err);
+    }
+
+    // Try to remove "1 - assigned" label
```

Comment:
The code is pretty expressive and IMO doesn't need some of these labels.

Reply:
Great feedback. Updated!